File: app/src/java/java/com/example/ama/viewer/presentation/list/adapter/MainListViewHolder.java
package com.example.ama.viewer.presentation.list.adapter;

import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import com.example.ama.viewer.R;

public class MainListViewHolder extends RecyclerView.ViewHolder {
    MainListViewHolder(@NonNull View itemView) {
        super(itemView);
    }

    void bind(String itemText) {
        ((TextView) itemView.findViewById(R.id.tv_item)).setText(itemText);
    }
}

File: app/src/kotlin/java/com/example/ama/viewer/presentation/list/mvp/adapter/MainListViewHolder.kt
package com.example.ama.viewer.presentation.list.mvp.adapter

import android.support.v7.widget.RecyclerView
import android.view.View
import kotlinx.android.synthetic.main.recycler_view_item.view.*

class MainListViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    fun bind(itemText: String) {
        itemView.tv_item.text = itemText
    }
}